components/app/dao-proposals/DaoProposalsList.vue

```diff
@@ -1,102 +1,107 @@
 <template>
-  <div
-    class="
-      grid grid-cols-1
-      md:grid-cols-2
-      lg:grid-cols-2
-      xl:grid-cols-4
-      2xl:grid-cols-4
-      gap-4
-      mt-10
-    "
-  >
-    <div v-for="daoProposal in daoProposals" :key="daoProposal._id">
-      <NuxtLink
-        :prefetch="false"
-        :to="`/projects/${beautifyProjectId(daoProposal.project)}`"
-      >
-        <div class="shadow rounded p-8 h-full">
-          <div class="flex">
-            <div class="mr-3">
-              <app-logo
-                class="inline-block"
-                :src="daoProposal.project.logo"
-                :alt="daoProposal.project.title"
-                :size="45"
-              />
+  <div class="mt-10">
+    <dao-proposals-response
+      :proposals-found="!!daoProposals.length"
```

Comment:
nitpick: !! can be removed

Reply:
The prop requires a boolean type. Falsy or truthy values throw errors accordingly.